content/_data/upgrades/2-387-1.adoc

```diff
@@ -0,0 +1,18 @@
+==== De-duplicate logging implementations
+
+Support for log rotation via `SIGALRM` has been removed.
```

Comment:
```suggestion
Support for log rotation via `SIGALRM` has been removed. The command-line argument `--daemon` has been removed.
```

Reply:
good catch @NotMyFault , thanks for pointing this out! I updated to include the line, I just did it in a separate commit to keep it as 1 sentence per line.